qiskit/backends/ibmq/ibmqbackend.py

```diff
@@ -141,18 +142,73 @@ def status(self):
                 "Couldn't get backend status: {0}".format(ex))
         return status
 
-    def jobs(self, limit=50, skip=0):
-        """Attempt to get the jobs submitted to the backend
+    def jobs(self, limit=50, skip=0, status=None, db_filter=None):
+        """Attempt to get the jobs submitted to the backend.
 
         Args:
             limit (int): number of jobs to retrieve
             skip (int): starting index of retrieval
+            status (None or JobStatus or str): only get jobs with this status,
+                where status is e.g. `JobStatus.RUNNING` or `'RUNNING'`
+            db_filter (dict): `loopback-based filter
+                <https://loopback.io/doc/en/lb2/Querying-data.html>`_.
+                This is an interface to a database ``where`` filter. Some
+                examples of its usage are:
+
+                Filter last five jobs with errors::
+
+                   job_list = backend.jobs(limit=5, status=JobStatus.ERROR)
+
+                Filter last five jobs with counts=1024, and counts for
+                states ``00`` and ``11`` each exceeding 400::
+
+                  cnts_filter = {'shots': 1024,
+                                 'qasms.result.data.counts.00': {'gt': 400},
+                                 'qasms.result.data.counts.11': {'gt': 400}}
+                  job_list = backend.jobs(limit=5, db_filter=cnts_filter)
+
+                Filter last five jobs from 30 days ago::
+
+                   past_date = datetime.datetime.now() - datetime.timedelta(days=30)
+                   date_filter = {'creationDate': {'lt': past_date.isoformat()}}
+                   job_list = backend.jobs(limit=5, db_filter=date_filter)
+
         Returns:
             list(IBMQJob): list of IBMQJob instances
+
+        Raises:
+            ValueError: status keyword value unrecognized
```

Comment:
Throwing ValueError in this case gives a lot semantic meaning, which could be very useful for our user's code, this is great, but in order to keep the policy of everything thrown from QISKit should inherit from `QISKitError`, I'd suggest adding even more semantic meaning by creating a new exception class here that inherits from `ValueError` and `IBMQBackendError`: 
```
IBMQBackendValueError(IBMQBackendError, ValueError):
   """ Value errors thrown within IBMQBackend """
   pass
```
This way, user's code could capture exceptions by `ValueError` or `QISKitError`.

Reply:
sounds good